.github/workflows/bump-golang.yml

```diff
@@ -18,13 +18,17 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        file: ['bump-golang.yml', 'bump-golang-7.17.yml']
+        file: ["bump-golang.yml", "bump-golang-7.17.yml"]
     steps:
       - uses: actions/checkout@v4
 
       - name: Install Updatecli in the runner
         uses: updatecli/updatecli-action@6b8881a17fc8038e884ec94ff72a49e8e8a4069f # v0.76.1
 
+      - uses: actions/setup-go@v5
+        with:
+          go-version-file: .go-version
```

Comment:
Will this cause any issues if the golang-bump happens afterwards? Let's say current version is 1.22.7 and the upcoming one is `1.22.8`, hence this step will point to `1.22.7` but the updatecli pipeline will move the version from `1.22.7` to `1.22.8`

Reply:
Good question. It's better to install the new version of go.

Reply:
Hence, this change will not work if that's the pre-req

Reply:
Updatecli can neither execute a specified target nor run a subset of targets. I came up with one solution that I don't like. 

1. run updatecli with `--commit false --push false`. It applies the changes to the target files.
2. install go with GHA using updated `.go-version`
3. run updatecli **again** without `--commit false --push false` and it pushes the changes.

Please check the updates. Do you have any ideas on how can we make it in one execution?

Reply:
First, can you explain the motivation for running `go mod tidy`? I have not seen the need to use it while running the same mechanism with other golang versions:

- https://github.com/elastic/beats/pull/41018
- https://github.com/elastic/beats/pull/40528/
- https://github.com/elastic/beats/pull/40114

> Updatecli can neither execute a specified target nor run a subset of targets.

As far as I see there is a field called `dependson`, see https://www.updatecli.io/docs/core/target/#_parameters:

> "dependson" allows to specify the order of execution of resources It accepts a list of rules like "(resourceType#)resourceId(:booleanOperator)" The resourceType is optional and can be one of "condition", "source" or "target"